mlflow/projects/__init__.py

```diff
@@ -188,7 +189,7 @@ def run(
     experiment_id=None,
     backend="local",
     backend_config=None,
-    use_conda=True,
+    use_conda=None,
```

Comment:
I changed the default value to `None` so we can tell whether a user specifies it or not. I'm checking if this is really ok.

Reply:
By default, both `use_conda` and `env_manager` are None, meaning mlflow automatically selects the right env manager.